src/main.js

```diff
@@ -0,0 +1,211 @@
+// factory function to create new board object
+var Board = function (pegs, discs) {
+  this.board = [];
+  this.pegs = pegs;
+  this.discs = discs;
+
+  this.startNewGame();
+}
+
+// add method to start a new game
+Board.prototype.startNewGame = function () {
+  this.board = [];
+  
+  for (var i = 0; i < this.pegs; i++) {
+    this.board.push([]);
+  }
+
+  for (var i = this.discs; i > 0; i--) {
+    this.board[0].push(i);
+  }
+
+  this.displayBoard();
+  return this.board;
+}
+
+// add method to update the board display board via HTML
+Board.prototype.displayBoard = function () {
+  gameDiv.innerText = '';
+
+  this.board.map(function (peg) {
+    var pegDiv = document.createElement('div');
+    pegDiv.setAttribute('class', 'peg');
+
+    var currentPeg = '--- ';
+    peg.map(function (disc) {
+      currentPeg += disc + ' ';
+    })
+    
+    pegDiv.innerText = currentPeg;
+    gameDiv.appendChild(pegDiv);
+  })
+}
+
+// add method to move a disc 
+Board.prototype.moveDisc = function (pegMoveFrom, pegMoveTo) {
+  var pegMoveFromIndex = pegMoveFrom - 1;
+  var pegMoveToIndex = pegMoveTo - 1;
+
+  // if they select a peg to move from that doesn't exist on the board
+  if (!this.board[pegMoveFromIndex]) {
+    return 'The peg you want to move a disc from doesn\'t exist! Board is still:';
+  // if they select a peg to move to that doesn't exist on the board  
+  } else if (!this.board[pegMoveToIndex]) {
+    return 'The peg you want to move a disc to doesn\'t exist! Board is still:';
+  // if they select a peg to move from that is empty
+  } else if (this.board[pegMoveFromIndex].length === 0) {
+    return 'There aren\'t any discs on that peg! Board is still:';
+  // if they select the same peg to move from and move to
+  } else if (this.board[pegMoveFromIndex] === this.board[pegMoveToIndex]) {
```

Comment:
The complexity of this function seems to be pretty high. I would recommend creating a helper function called determineResponse to return the correct response.